VAMobile/src/screens/BenefitsScreen/Letters/LettersListScreen.tsx

```diff
@@ -127,24 +128,31 @@ function LettersListScreen({ navigation }: LettersListScreenProps) {
     return letterButton
   })
 
-  const errorCheck = getLettersError || getUserAuthorizedServicesError
-  const noLettersCheck = !userAuthorizedServices?.lettersAndDocuments || !letters || letters.length === 0
-
   return (
     <FeatureLandingTemplate
       backLabel={t('letters.overview.title')}
       backLabelOnPress={navigation.goBack}
       title={t('letters.overview.viewLetters')}
       {...testIdProps('Letters-list-page')}>
-      {loading || loadingUserAuthorizedServices ? (
+      {!lettersNotInDowntime ? (
+        <ErrorComponent screenID={ScreenIDTypesConstants.LETTERS_LIST_SCREEN_ID} />
+      ) : loading || loadingUserAuthorizedServices ? (
         <LoadingComponent text={t('letters.list.loading')} />
-      ) : errorCheck || !lettersNotInDowntime ? (
+      ) : getUserAuthorizedServicesError ? (
+        <ErrorComponent
+          screenID={ScreenIDTypesConstants.LETTERS_LIST_SCREEN_ID}
+          error={getUserAuthorizedServicesError}
+          onTryAgain={refetchAuthServices}
+        />
```

Comment:
Same here and the other files that were updated in this PR. We shouldn't be displaying the error component when the authorized services endpoint has failed

Reply:
same as my comment above, I disagree with this statement and we may need product to weigh in. Cause what will happen is the not authorized screen will then display instead causing a worse experience since that does not have a refresh